src/components/lyne-panel/lyne-panel.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Component,
+  h,
+  Prop
+} from '@stencil/core';
+import { InterfacePanelAttributes } from './lyne-panel.custom.d';
+
+@Component({
+  shadow: true,
+  styleUrl: 'lyne-panel.scss',
+  tag: 'lyne-panel'
+})
+
+export class LynePanel {
+
+  /** The text to show in the panel */
+  @Prop() public text!: string;
+
+  /** The text to use as button text */
+  @Prop() public buttonText!: string;
+
+  /** Id which is sent in the click event payload for the button*/
+  @Prop() public eventId?: string;
+
+  /** The tag to use for the text element */
+  @Prop() public tag?: InterfacePanelAttributes['tag'] = 'p';
+
+  public render(): JSX.Element {
+    const TAGNAME = this.tag;
+
+    return (
+      <div class='lyne-panel'>
```

Comment:
We will also have to re-add the "Paid Content" module in here at a later stage.

Reply:
As discussed, i propose to not integrate that for the moment until we know if we will be using hotel stars in the panel in the future and until we have an appropriate design for it